dts/arm/st/l1/stm32l152Xc.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Abel Sensors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
This commit can be extracted from this PR and pushed as standalone PR.
But, please take into account related comments